projects/packages/forms/tests/php/contact-form/Contact_Form_Test.php

```diff
@@ -1105,7 +1143,7 @@ public function assertValidCheckboxField( $html, $attributes ) {
 		$this->assertInstanceOf( DOMElement::class, $label );
 		$this->assertInstanceOf( DOMElement::class, $input );
 
-		$this->assertEquals( $label->getAttribute( 'class' ), 'grunion-field-label ' . $attributes['type'], 'label class doesn\'t match' );
+		$this->assertEquals( $label->getAttribute( 'class' ), 'wp-block-jetpack-option grunion-field-label ' . $attributes['type'], 'label class doesn\'t match' );
```

Comment:
Is this correct?

Reply:
I believe so. The `render_checkbox_field` code has been updated to include `wp-block-jetpack-option` here: https://github.com/Automattic/jetpack/blob/f48de08ec04a90b0c7a4511689309e740a11e1f7/projects/packages/forms/src/contact-form/class-contact-form-field.php#L883

So this update looks correct to me.